src/clients/catalog.ts

```diff
@@ -18,7 +16,28 @@ export class Catalog extends JanusClient {
     const metric = 'catalog-getSkuMetric'
     const token = getAuthToken(this.context, authMethod)
 
-    return this.http.get<OrderFormConfiguration>(catalogRouteEndpoint(skuId), {
+    return this.http.get<OrderFormConfiguration>(this.routes.getSkuById(skuId), {
+      headers: token
+        ? {
+            VtexIdclientAutCookie: token,
+          }
+        : {},
+      metric,
+      tracing: createTracing(metric, tracingConfig),
+    })
+  }
+
+  // eslint-disable-next-line max-params
```

Comment:
What do you think about named parameters?

```typescript
{ sellerId, skuId, authMethod = AUTH_TOKEN }: ChangeNotificationArgs
```

Don't remember if we already tested this